yb-voyager/cmd/root.go

```diff
@@ -120,3 +135,37 @@ func validateExportDirFlag() {
 		exportDir = strings.TrimRight(exportDir, "/")
 	}
 }
+
+func lockExportDir() {
+	lockfileName, err := filepath.Abs(filepath.Join(exportDir, ".lockfile.lck"))
+	if err != nil {
+		fmt.Println("Unable to create an absolute path for the export-dir lockfile")
+		os.Exit(1)
+	}
+
+	lock, err = lockfile.New(lockfileName)
+	if err != nil {
+		fmt.Printf("Failed to init lock on export-dir. Unable to create lockfile %v. Reason: %v", lockfileName, err)
+		os.Exit(1)
```

Comment:
Use `utils.ErrExit(...)`.
`"failed to create lock file %q: %v", lockFileName, err`